tools/data/textdet/mtwi_converter.py

```diff
@@ -0,0 +1,201 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import math
+import os
+import os.path as osp
+
+import cv2
+import mmcv
+from PIL import Image
+
+from mmocr.utils import convert_annotations
+
+
+def collect_files(img_dir, gt_dir, ratio):
+    """Collect all images and their corresponding groundtruth files.
+    Args:
+        img_dir (str): The image directory
+        gt_dir (str): The groundtruth directory
+        ratio (float): Split ratio for val set
+
+    Returns:
+        files (list): The list of tuples (img_file, groundtruth_file)
+    """
+    assert isinstance(img_dir, str)
+    assert img_dir
+    assert isinstance(gt_dir, str)
+    assert gt_dir
+    assert isinstance(ratio, float)
+    assert ratio < 1.0, 'val_ratio should be a float between 0.0 to 1.0'
+
+    ann_list, imgs_list = [], []
+    for ann_file in os.listdir(gt_dir):
+        img_file = osp.join(img_dir, ann_file.replace('txt', 'jpg'))
+        # This dataset contains some images obtained from .gif,
+        # which cannot be loaded by mmcv.imread(), convert them
+        # to RGB mode.
```

Comment:
Is skipping gif files a common practice? If not, we should try another backend like Pillow to load gif files, and try to save them in the general format that OpenCV can read. Otherwise, the dataset is incomplete.

https://mmcv.readthedocs.io/en/latest/api.html#mmcv.image.imread